app/models/submission.rb

```diff
@@ -0,0 +1,18 @@
+class Submission < ApplicationRecord
+  SUBMISSION_STATUSES = [
+    "Received",
+    "In-progress",
+    "Declined",
+    "Accepted",
+    "Completed",
+    "Withdrawn",
+    "Editable",
+  ]
+  belongs_to :user
+  has_many :submission_packets, dependent: :destroy
+  has_many :poems, through: :submission_packets
+
+  validates_presence_of :title 
+  validates :status, presence: true, inclusion: { in: SUBMISSION_STATUSES }
+  validates_presence_of :submitted_to
```

Comment:
`validates` is the new (and preferred) way of defining validations. Looks like you used it on line `16`, any reason you didn't do it here?